packages/youtube/commands/youtube/backsync_comments.ts

```diff
@@ -15,21 +15,21 @@ limitations under the License.
 */
 
 import { google } from 'googleapis';
-import * as yargs from 'yargs';
+import * as Yargs from 'yargs';
```

Comment:
I think package names are more commonly starting with lower case, curious why you made this upper case?

Reply:
We use "yargs" as a function parameter later on in the function.  This causes lint errors.  

Will be deleting this code soon anyway - not sure why I even made the change except I hate seeing the red underlines in my IDE:-P

Reply:
hehe. better to rename the argument I think then.